src/hmem_ze.c

```diff
@@ -1130,12 +1130,19 @@ int ze_hmem_host_unregister(void *ptr)
 	return FI_SUCCESS;
 }
 
-static int ze_get_dma_buf_fd(const void *addr)
+int ze_hmem_get_dmabuf_fd(void *addr, uint64_t size, int *fd, uint64_t *offset)
 {
+	int ret;
 	ze_result_t ze_ret;
 	ze_memory_allocation_properties_t mem_props = {};
 	ze_device_handle_t device_ptr;
 	ze_external_memory_export_fd_t export_fd = {};
+	void *base_addr;
+	uint64_t total_size;
+
+	ret = ze_hmem_get_base_addr(addr, size, &base_addr, &total_size);
```

Comment:
Does this necessarily be page aligned base addr?

Reply:
Yes, that's true for Ze.

Reply:
Gotcha, I just learned it's the same for neuron. We may need to add such get_base_addr op for neuron in the future to make it consistent